src/main/charts/confluence/values.yaml

```diff
@@ -744,6 +744,11 @@ synchrony:
   # This will result in a separate StatefulSet and Service to be created for Synchrony.
   # If disabled, then collaborative editing will be disabled in Confluence.
   enabled: false
+
+  # -- Custom annotations that will be applied to all Synchrony pods.
```

Comment:
Should we mention that if not defined, `confluence.podAnnotations` will be used?

Reply:
Yes, it's there in the next line